show/plugins/cisco-8000.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+#########################################################
+# Copyright 2021 Cisco Systems, Inc.
+# All rights reserved.
+#
+# CLI Extensions for show command
+#########################################################
+
+try:
+    import click
+    import yaml
+    from show import platform
+    import utilities_common.cli as clicommon
+except ImportError as e:
+    raise ImportError("%s - required module not found" % str(e))
+
+PLATFORM_PY = '/opt/cisco/bin/platform.py'
+
+# Platform specific version invoked by generic show.version
+def version(verbose):
+    filespec = '/opt/cisco/etc/build_info.yaml'
+    with open(filespec) as f:
+        build_info = yaml.safe_load(f)
+    for d in build_info.get('sdk_versions', []):
+        for k,v in d.items():
+            click.echo('{}: {}'.format(k,v))
+
+@click.command()
+def inventory():
+    """ Show platform inventory information """
+    cmd = 'inventory.py show'
```

Comment:
don't see these file inventory.py added ? Is it also present in /opt/cisco/bin/ ?

Reply:
For the inventory, a better approach will be define something new in sonic_platform, define API's and use those API's here instead of calling platform specific scripts.
Ref: https://github.com/Azure/sonic-utilities/blob/9b7c58b678998c2dbab1f4d5ee9ec6a9294ab666/show/platform.py#L13

Reply:
ack. Moved inventory() method to show/platform.py. A call to inventoryshow will be made from platform.py. For now, the inventoryshow is a python module sitting under /usr/local/bin/inventoryshow. this is built as part of cisco's platform code and placed under the target path (/usr/local/bin).

Reply:
My idea was to create a inventory base class in https://github.com/Azure/sonic-platform-common and implement that in Cisco 8800 platform  ( similar to as it is done here https://github.com/Azure/sonic-buildimage/tree/master/platform/broadcom/sonic-platform-modules-dell/s6100/sonic_platform for fan/psu/sfp etc. )

But if that is a bigger task, we should keep this CLI in show/plugins/cisco-8000.py itself as before. The inventoryshow script wont be present for all other platforms.